cross/libgpg-error/Makefile

```diff
@@ -11,6 +11,13 @@ HOMEPAGE = http://www.gnupg.org/related_software/libgpg-error/
 COMMENT  = Libgpg-error is a small library that defines common error values for all GnuPG components
 LICENSE  = LGPL
 
+PRE_CONFIGURE_TARGET = preConfigureTarget
 GNU_CONFIGURE = 1
 
 include ../../mk/spksrc.cross-cc.mk
+
+.PHONY: preConfigureTarget
+preConfigureTarget:
+	cp $(WORK_DIR)/$(PKG_DIR)/src/syscfg/lock-obj-pub.arm-unknown-linux-gnueabi.h $(WORK_DIR)/$(PKG_DIR)/src/syscfg/lock-obj-pub.linux-gnueabi.h
```

Comment:
@adrien-delhorme https://github.com/SynoCommunity/spksrc/pull/1680 has a better way of handling the libgpg-error changes. That PR will be merged in the next couple of days. The easiest for you to do is to remove all libgpg-error related changes in this PR.